infrastructure/TuDa.CIMS.MigrationService/Worker.cs

```diff
@@ -79,15 +80,13 @@ private async Task SeedDataAsync(CIMSDbContext dbContext, CancellationToken canc
     {
         Randomizer.Seed = new Random(12345);
 
-        var room = new RoomFaker().Generate();
-
-        var chemical = new ChemicalFaker(room).Generate();
-        var solvent = new SolventFaker().Generate();
-        var gasCylinder = new GasCylinderFaker().Generate();
-        var consumable = new ConsumableFaker(room).Generate();
+        (Consumable consumable, Chemical chemical) = await SeedAssetItem(
+            dbContext,
+            cancellationToken
+        );
 
-        var professor = new PersonFaker<Professor>(null!).Generate();
-        var students = new PersonFaker<Student>(null!).GenerateBetween(5, 5);
+        var professor = new PersonFaker<Professor>().Generate();
+        var students = new PersonFaker<Student>().GenerateBetween(5, 5);
```

Comment:
What does "null!" mean as a parameter and why don't we need that anymore?

Reply:
`null` means `null` and `!` is used to suppress null warnings. Here, the default value for the `PersonFaker` parameter Working Group is null, so null as parameter is not needed. I do not know why I left in there last time.